src/wp-admin/includes/nav-menu.php

```diff
@@ -351,6 +351,7 @@ function wp_nav_menu_item_link_meta_box() {
 				type="text"<?php wp_nav_menu_disabled_check( $nav_menu_selected_id ); ?>
 				class="code menu-item-textbox form-required" placeholder="https://"
 			/>
+			<span id="custom-url-error" class="error-message" style="display: none;">Invalid URL.</span>
```

Comment:
This also needs to be a translatable string; and, similarly, should use text that better reflects the real validation.

Preferably, the error message reported to screen readers should be the same as the one displayed in the input.